ansible_collections/arista/avd/roles/eos_designs/templates/network_services/router-bgp-vlans.j2

```diff
@@ -1,21 +1,24 @@
 {# tenant router bgp evpn configuration- MAC-VRF VLAN-Based #}
   vlans:
-{% if vxlan_vlan_aware_bundles == false %}
+{% set evpn_vlan_aware_bundles = vxlan_vlan_aware_bundles | arista.avd.default(evpn_vlan_aware_bundles, false) %}
+{% if evpn_vlan_aware_bundles == false %}
```

Comment:
IMO, ```if not evpn_vlan_aware_bundles```

Reply:
TBH I would change this to != true just to be exactly opposite of the logic in bundles templates.
Or change both as you propose, but as noted above, we don't do that elsewhere.

Reply:
!= true would be the best IMO, since as it is now, an incorrect evpn_vlan_aware_bundles value of strings "true" or "false" or "foo" or similar unsupported value would not match either of the templates. =! true would make sure it falls back to use vlans even if the variable is defined in an invalid way.

Reply:
I have changed from != true after comment from emil as well.